galley/pkg/config/analysis/analyzers/util/config.go

```diff
@@ -40,6 +40,7 @@ func IsSystemNamespace(ns resource.Namespace) bool {
 	return ns == "kube-system" || ns == "kube-public"
 }
 
-func IsIstioSystemNamespace(ns resource.Namespace) bool {
-	return ns == "istio-system"
+func IsIstioControlPlane(labels map[string]string) bool {
```

Comment:
IMO it makes the most sense to pass the resource as the arg here. For instance, if we ever make the logic for deciding what's in the control plane more complex, calling code shouldn't need to know.